workbench/tests/binary_tests/puppet.test.js

```diff
@@ -10,8 +10,6 @@ import { spawn, spawnSync } from 'child_process';
 
 import rimraf from 'rimraf';
 import puppeteer from 'puppeteer-core';
-import { getDocument, queries, waitFor } from 'pptr-testing-library';
```

Comment:
We had to stop using `pptr-testing-library` because it seems to be no longer maintained. And the latest release depends on a different version of `testing-library/dom` than does `testing-library/react`. And having two different versions of that dependency caused problems.